src/main/java/com/faforever/api/security/AuditService.java

```diff
@@ -16,8 +16,8 @@ public AuditService(UserSupplier userSupplier) {
   }
 
   public void logMessage(String message) {
-    final FafUserDetails fafUserDetails = userSupplier.get();
-    String extendedMessage = MessageFormat.format("{0} [invoked by User ''{1}'' with id ''{2}'']", message, fafUserDetails.getUsername(), fafUserDetails.getId());
+    final FafAuthenticationToken fafAuthenticationToken = userSupplier.get();
+    String extendedMessage = MessageFormat.format("{0} [invoked by User  with id ''{2}'']", message, fafAuthenticationToken.getUserId());
```

Comment:
@bukajsytlos  I'm open for ideas how to keep the name but I think it would imply a db lookup fo the name everytime, which I wanted to avoid...

Reply:
Don't know about keep a name, but point of my comment was that since there are now only two parameters index 0 and 1, {2} would reference nonexisting value

Reply:
nvm, github showed me outdated code

Reply:
regarding name, do I understand it correctly, that name could/should be part of the user service, and it could be part of jwt token?

Reply:
yes we could pit it into the extension

Reply:
Done